src/lib/Content/Form/Provider/NonLocalizedGroupedContentFormFieldsProvider.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\ContentForms\Content\Form\Provider;
+
+use Ibexa\Contracts\ContentForms\Content\Form\Provider\GroupedContentFormFieldsProviderInterface;
+use Ibexa\Core\Helper\FieldsGroups\FieldsGroupsList;
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+
+class NonLocalizedGroupedContentFormFieldsProvider implements GroupedContentFormFieldsProviderInterface, TranslationContainerInterface
+{
+    /** @var \Ibexa\Core\Helper\FieldsGroups\FieldsGroupsList */
+    private $fieldsGroupsList;
```

Comment:
Prefer typed properties over annotations, when dealing with PHP 7.4+ and no BC.
```suggestion
    private FieldsGroupsList $fieldsGroupsList;
```